docs/docs/guidelines/release-process.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Release Process
+sidebar_label: Release Process
+---
+
+Since the [Airy](https://github.com/airyhq/airy) repository is a mono-repo, the
+following document provides guidelines about the release process of the various
+projects the repository contains.
+
+To release new software, we use a combination of conventions for git tags and branches. The high-level process looks like this:
+
+- We create a release branch
+- We tag the new branch with new version
```

Comment:
Does this mean to create and push a git tag? 🤔 In my understanding, we merge the release branch to main and then create and push the tag, but I may be mistaken.

Reply:
Ah that's described below.

Reply:
yeah to be fair I'm unsure about this initial paragraph... may be worth removing all together? It may confused readers